### DISCUSSION TITLE
PowerNukkit v1.5.2.1-PN

### DISCUSSION THREAD

**Discussion by Polyana-Fontes** [PowerNukkit/PowerNukkit#1275] [Show and tell]
## [1.5.2.1-PN] - 2021-12-21 ([Check the milestone](https://github.com/PowerNukkit/PowerNukkit/milestone/30?closed=1))
This is a critical security update to fix [CVE-2021-45105](https://github.com/advisories/GHSA-p6xc-xr62-6r2g) and [CVE-2021-44228](https://github.com/advisories/GHSA-jfh8-c2jp-5v3q)

### CRITICAL SECURITY FIX
- #1266, #1270 Changed Log4J library from `2.13.3` to `2.17.0`

## Pterodactyl
The Pterodactyl Eggs didn't change, it's being reposted here for easy access.

[1.5.2.1-PN]: https://github.com/PowerNukkit/PowerNukkit/compare/v1.5.2.0-PN...v1.5.2.1-PN

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerNukkit/PowerNukkit/releases/tag/v1.5.2.1-PN'>PowerNukkit v1.5.2.1-PN</a>.</em>

**Comment by ReiyansCN** [comment]
It's great！Thank you for your work about PowerNukkit

**Comment by Minearm-RPM** [comment]
还行吧，总体水平还不错的，但是缺点就是WAC容易误判，令人烦不胜烦
